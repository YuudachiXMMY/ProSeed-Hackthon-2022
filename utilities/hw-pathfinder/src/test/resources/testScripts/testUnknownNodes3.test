# Test output with unknown input nodes

CreateGraph graph

AddNode graph A

FindPath graph B C