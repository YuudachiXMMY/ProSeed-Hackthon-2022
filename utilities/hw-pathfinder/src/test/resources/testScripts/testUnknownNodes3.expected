# Test output with unknown input nodes

created graph graph

added node A to graph

unknown node B
unknown node C